crates/blockless-drivers/src/error.rs

```diff
@@ -33,7 +33,7 @@ impl std::fmt::Display for ErrorKind {
     }
 }
 
-#[derive(Debug)]
+#[derive(Debug, PartialEq)]
 pub enum HttpErrorKind {
     InvalidDriver,
     InvalidHandle,
```

Comment:
nit:
If `PartialEq` is only required for dev/tests, then does not need to be derived for non-test envs.

```suggestion
#[derive(Debug)]
#[cfg_attr(test, derive(PartialEq))]
pub enum HttpErrorKind {
    InvalidDriver,
    InvalidHandle,
```